tests/at_onboarding_cli_functional_tests/test/enrollment_test.dart

```diff
@@ -24,19 +24,32 @@ void main() {
   AtSignLogger.root_level = 'WARNING';
   String storageDir = 'test/storage';
```

Comment:
No material changes in this file, just pulled the atSign variables out into main scope so it's easier to change them